roles/openshift_examples/templates.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+EXAMPLES_BASE=$(pwd)/files/examples
```

Comment:
Can we make this absolute path instead? I ask because when we start copying things back into place we're in the temp dir.